src/EnergyPlus/NodeInputManager.cc

```diff
@@ -109,8 +110,8 @@ namespace NodeInputManager {
 		NumCheckNodes = 0;
 		MaxCheckNodes = 0;
 		NodeVarsSetup = false;
-		NodeWetBulbRepReq.deallocate();
```

Comment:
I removed it because at line 1114 of NodeInputManager a lot of arrays in addition to this one get allocated if CalcMoreNodeInfoMyOneTimeFlag is true, which is set true in clear_state at line 102. Since there were many other arrays at line 1114 that were not in clear_state, I figured this lone wolf wasn't needed in the clear_state function. I assume they get reallocated to the correct size at line 1114. No need for more than is needed in clear_state. The test for this is unit tests and they seem to be passing without this line.